# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/ImageLoader.cpp
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b layout/style/ImageLoader.cpp
--- a/layout/style/ImageLoader.cpp	Wed Aug 21 10:29:14 2019 +0000
+++ b/layout/style/ImageLoader.cpp	Wed Aug 21 12:52:19 2019 +0000
@@ -790,20 +790,6 @@
   return NS_OK;
 }
 
-void ImageLoader::FlushUseCounters() {
-  MOZ_ASSERT(NS_IsMainThread());
-
-  for (auto iter = mRegisteredImages.Iter(); !iter.Done(); iter.Next()) {
-    if (imgRequestProxy* request = iter.Data()) {
-      nsCOMPtr<imgIContainer> container;
-      request->GetImage(getter_AddRefs(container));
-      if (container) {
-        static_cast<image::Image*>(container.get())->ReportUseCounters();
-      }
-    }
-  }
-}
-
 bool ImageLoader::ImageReflowCallback::ReflowFinished() {
   // Check that the frame is still valid. If it isn't, then onload was
   // unblocked when the frame was removed from the FrameSet in